recipe/meta.yaml

```diff
@@ -18,10 +18,12 @@ source:
     # https://github.com/conda-forge/staged-recipes/pull/199
     #
     - test_Makefile.in.patch
+    # Disable shared Fortran API building on OSX
+    - osx_configure.patch  # [osx]
 
 build:
-  number: 10
-  skip: True  # [py==36]
+  number: 11
+  skip: True  # [win and py==36]
```

Comment:
Will give that a shot - left it as is due to ```py36``` not being listed on https://conda.io/docs/building/meta-yaml.html